src/library/Uno.Toolkit.Material/Styles/Controls/v2/TabBar.xaml

```diff
@@ -805,7 +805,6 @@
                                 <VisualState x:Name="Enabled" />
                                 <VisualState x:Name="Disabled">
                                     <VisualState.Setters>
-                                        <Setter Target="LayoutRoot.Background" Value="{ThemeResource NavigationTabBarBackgroundDisabled}" />
```

Comment:
That was wrongly added. The background doesn't change in Disabled state and it was crashing on Windows, since `NavigationTabBarBackgroundDisabled` doesn't exist.